flows-advanced/prefect-sqlalchemy/etl_with_sql_alchemy.py

```diff
@@ -0,0 +1,61 @@
+import os
+import pandas as pd
+from prefect import flow, task
+from prefect.tasks import task_input_hash
+from datetime import timedelta
+from prefect_sqlalchemy import SqlAlchemyConnector
+
+
+@task(log_prints=True, tags=["extract"], cache_key_fn=task_input_hash, cache_expiration=timedelta(days=1))
+def extract_data(url: str):
+    # the backup files are gzipped, and it's important to keep the correct extension
+    # for pandas to be able to open the file
+    if url.endswith('.csv.gz'):
+        csv_name = 'yellow_tripdata_2021-01.csv.gz'
+    else:
+        csv_name = 'output.csv'
+    
+    os.system(f"wget {url} -O {csv_name}")
+
+    df_iter = pd.read_csv(csv_name, iterator=True, chunksize=100000)
```

Comment:
I don't think it's necessary to wget first. `pd.read_csv` can read from URLs directly
```suggestion
    df = pd.read_csv(url, compression="gzip")
```